build.gradle

```diff
@@ -79,6 +93,15 @@ dependencies {
       translateCompile files('lib/json-simple-1.1.1.jar')
       translateCompile 'joda-time:joda-time:2.9.3'
       translateCompile 'org.json:json:20140107'
+
+    // 'compile' dep also needed to run 'gradle jmh' for some reason,
```

Comment:
I meant it to stand for 'dependency' (I updated the comment to hopefully be more readable)